pysph/examples/gas_dynamics/wc_blastwave.py

```diff
@@ -36,8 +38,18 @@ def initialize(self):
         self.hdx = 1.5
         self.n_particles = 1000
 
+    def add_user_options(self, group):
+        group.add_argument(
+            "--hdx", action="store", type=float,
+            dest="hdx", default=1.5,
+            help="Ratio h/dx."
+        )
+        group.add_argument("--nparticles", action="store", type=float,
```

Comment:
Can this be `--np` instead? Or perhaps `-n /--n-particles`?

Reply:
Examples `accuracy_test_2d` and `acoustic-wave` have been using `--nparticles`. Should I change those as well?